assets/css/sidebar.css

```diff
@@ -245,6 +245,7 @@
   font-size: 0.9em;
   line-height: 1.8em;
   color: var(--sidebarSubheadings);
+  margin-top: 10px;
```

Comment:
The grouping functionality uses 1.5em. Should we use the same?

```suggestion
  margin-top: 1.5em;
```

Reply:
I'm find with that if you'd prefer. I personally think it errs on the side of too much margin, but I don't feel strongly. Here's what that looks like:

![image](https://github.com/elixir-lang/ex_doc/assets/503938/c42b93ad-21a2-4601-af97-fd6c6c645df9)

Reply:
Looking at the grouping functionality: that font size is `.8em` whereas the module nesting uses `.9em`, so `1.5em` margin ends up being a bit larger for module nesting. To make them consistent, we could update module nesting to use `.8em` and `1.5em` margin.

Reply:
(Sorry for the repeated messages.) Looking at it now, bumping the nested modules down to `.8em` seems a bit too small. The other grouping uses ALL CAPS, so it's more readable at that smaller size.

I'll ultimately defer to you.